statusnet/statusnet.php

```diff
@@ -1050,7 +1050,7 @@ function statusnet_createpost(App $a, $uid, $post, $self, $create_user, $only_ex
 		$own_url = DI::pConfig()->get($uid, 'statusnet', 'own_url');
 
 		if ($content->user->id == $own_url) {
-			$self = DBA::selectFirst([], ['self' => true, 'uid' => $uid]);
+			$self = DBA::selectFirst('', ['self' => true, 'uid' => $uid]);
```

Comment:
This call is missing a table name. The empty array means `*` for the field selection parameter.

Reply:
I added the `*` there.